stock_partner_delivery_window/models/res_partner.py

```diff
@@ -98,61 +95,6 @@ def is_in_delivery_window(self, date_time):
                     return True
         return False
 
-    def _get_delivery_time_format_string(self):
-        return _("From %s to %s")
-
-    def get_delivery_time_description(self):
```

Comment:
Why have you dropped those 2 methods ?

Reply:
@jbaudoux i removed them because i found no place call them.

Reply:
this is used in a module of cosanum v14, better keep it indeed as likely to be needed when cosanum goes v16